diff -rup modulesa/tripal_analysis_blast/tripal_analysis_blast.install modules/tripal_analysis_blast/tripal_analysis_blast.install
--- modulesa/tripal_analysis_blast/tripal_analysis_blast.install	2015-06-02 04:03:12.000000000 +0200
+++ modules/tripal_analysis_blast/tripal_analysis_blast.install	2016-02-23 13:26:15.268914781 +0100
@@ -436,25 +436,36 @@ function tripal_analysis_blast_update_72
   try {
     db_query('
       ALTER TABLE chado.blast_hit_data
-      DROP CONSTRAINT IF EXISTS blast_hit_data_db_id_fkey CASCADE
-    ');
+      DROP CONSTRAINT blast_hit_data_db_id_fkey CASCADE
+    '); } catch (\PDOException $e) {}
+ try {
     db_query('
       ALTER TABLE chado.blast_hit_data
-      DROP CONSTRAINT IF EXISTS blast_hit_data_analysisfeature_id_fkey CASCADE
-    ');
+      DROP CONSTRAINT blast_hit_data_analysisfeature_id_fkey CASCADE
+    '); } catch (\PDOException $e) {}
+ try {
     db_query('
       ALTER TABLE chado.blast_hit_data
-      DROP CONSTRAINT IF EXISTS blast_hit_data_analysis_id_fkey CASCADE
-    ');
+      DROP CONSTRAINT blast_hit_data_analysis_id_fkey CASCADE
+    ');} catch (\PDOException $e) {}
+ try {
     db_query('
       ALTER TABLE chado.blast_hit_data
-      DROP CONSTRAINT IF EXISTS blast_hit_data_feature_id_fkey CASCADE
-    ');
+      DROP CONSTRAINT blast_hit_data_feature_id_fkey CASCADE
+    ');} catch (\PDOException $e) {}
+ try {
     db_query('
       ALTER TABLE chado.blast_hit_data
-      DROP CONSTRAINT IF EXISTS blast_hit_data_blast_org_id_fkey CASCADE
+      DROP CONSTRAINT  blast_hit_data_blast_org_id_fkey CASCADE
     ');
+}

+catch (\PDOException $e) {
+    $error = $e->getMessage();
+
+ //   throw new DrupalUpdateException('Failed to update foriegn key: '. $error);
+  }
+try {
     db_query('
       ALTER TABLE chado.blast_hit_data
       ADD CONSTRAINT blast_hit_data_db_id_fkey

